docs/tutorials/multiplayergamearhitectureintroduction.md

```diff
@@ -0,0 +1,12 @@
+---
+id: multiplayergamearhitectureintroduction
+title: Multiplayer Game Architecture
```

Comment:
Architecture vs Model, which term do we use?

Reply:
Ill go with Architecture for now and if users dont like it ill change